src/DynamoUtilities/Md2Html.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Utilities
+{
+    public class Md2Html
+    {
+        private readonly Process process = new Process();
+        private readonly bool started;
+        /// <summary>
+        /// Constructor
+        /// Start the CLI tool and keep it around
+        /// </summary>
+        public Md2Html()
+        {
+            ProcessStartInfo startInfo = new ProcessStartInfo
+            {
+                CreateNoWindow = true,
+                RedirectStandardOutput = true,
+                RedirectStandardInput = true,
+
+                UseShellExecute = false,
+                Arguments = @"",
+                FileName = GetToolPath()
+            };
+
+            process.StartInfo = startInfo;
+            try
+            {
+                process.Start();
+                AppDomain.CurrentDomain.ProcessExit += new EventHandler(ProcessExit);
+                started = true;
+            }
+            catch(Win32Exception)
+            {
+                // Do nothing
+            }
+        }
+
+        /// <summary>
+        /// Destructor
+        /// Kill the CLI tool, if still running
+        /// </summary>
+        ~Md2Html()
```

Comment:
finalizers.... hmmm
I think IDisposable has a better chance of being deterministic and controllable in testing scenarios - 
just some thoughts:
https://www.viva64.com/en/b/0437/